.docker/aiida-core-dev/aiida-clone-and-install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd ~ || exit
+git clone https://github.com/aiidateam/aiida-core.git --origin upstream
+cd aiida-core || exit
+pip install --user -e ."[pre-commit,atomic_tools,docs,rest,tests,tui]" tox
+cd || exit
```

Comment:
```suggestion
pip install --user -e "/home/aiida/aiida-core/[pre-commit,atomic_tools,docs,rest,tests,tui]" tox
```

I think this works without `cd`.

Reply:
You can also set `REPO_PATH=/home/aiida/aiida-core` and use `REPO_PATH`.

Reply:
Is there a reason why you made this change?